Source/MonoGame.Extended/Maps/Tiled/TiledMapReader.cs

```diff
@@ -52,6 +53,8 @@ protected override TiledMap Read(ContentReader reader, TiledMap existingInstance
                         tileSetTile.CreateTileSetTileFrame(order: k, tileId: frameId, duration: reader.ReadInt32());
                     }
                     ReadCustomProperties(reader, tileSetTile.Properties);
+                    
+                    tileSetTile.ObjectGroups.AddRange(ReadObjectGroups(reader));
```

Comment:
Are you sure you didn't miss a file when committing? I'm getting a compile error on this line.